src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ToolTipTests.cs

```diff
@@ -782,6 +782,36 @@ public void ToolTip_ToString_Invoke_ReturnsExpected()
         Assert.Equal("System.Windows.Forms.ToolTip InitialDelay: 500, ShowAlways: False", toolTip.ToString());
     }
 
+#pragma warning disable WFO5001
+    [WinFormsFact]
+    public void ToolTip_DarkMode_GetColors_ReturnsExpected()
+    {
+        if (SystemInformation.HighContrast)
+        {
+            // We don't run this test in HighContrast mode.
+            return;
+        }
+
+        SystemColorMode colorMode = Application.ColorMode;
+        Application.SetColorMode(SystemColorMode.Dark);
+
+        using SubToolTip toolTip = new();
+
+        Assert.NotEqual(IntPtr.Zero, toolTip.Handle); // A workaround to create the toolTip native window Handle
```

Comment:
nit: let's use fluent assertions for the test
```suggestion
        toolTip.Handle.Should().NotBe(IntPtr.Zero); // A workaround to create the toolTip native window Handle
```